docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+```xml
+
+<concurrent-kickoff/>       (global cycle 1st increment recorded)
+
+<exclusive-start/>          (STW pause starts)
+
+<cycle-start/>              (global cycle starts)
+
+<exclusive-end/>            (STW pause ends)
+
+
+(global cycle 2nd increment running concurrently)
+
+
+<exclusive-start/>          (STW for partial GC cycle starts)
+
+
+...                         (partial GC cycle starts and completes)
+
+
+<exclusive-end/>            (STW for partial GC cycle ends)
+
+<exclusive-start/>          (STW pause starts)
+
+<concurrent-global-final/>  (global cycle final increment recorded)
+
+<gc-start/>                 (global cycle final increment starts)
+
+<allocation-stats/>
+
+<mem-info>                  (memory status before operations)
+
+<mem></mem>                 (status of different types of memory)
+
+</mem-info>
+
+</gc-start>
+
+<gc-op> “type=rs-scan"</gc-op>        (roots scanning completed)
+
+<gc-op>”type=card-cleaning" </gc-op>  (card cleaning completed)
+
+<gc-op> “type=mark”</gc-op>           (mark operation completed)
+
+<gc-op> “type=classunload”</gc-op>    (class unload operation completed)
+
+<gc-op ”type=sweep” />                (sweep operation completed)
+
+<gc-end>                     (global cycle final increment ends)
+
+<mem-info>                   (memory status after operations)
+
+<mem></mem>                  (status of different types of memory)
+
+</mem-info>
+
+</gc-end> 
+
+</cycle-end>                 (global cycle ends)
+
+<exclusive-end>              (STW pause ends)
+
+<exclusive-start>            (STW pause starts)
+...
+
+```
+
+The first activity in the cycle is recorded by a `<concurrent-kickoff>` element, which records the start of the first of three increments that make up a `gencon` Global GC cycle. The `<concurrent-kickoff>` element records:
+
+- the reason why the GC cycle was triggered. For a `gencon` global cycle, the cycle is triggered when the amount of available memory reaches a particular threshold value.
+- The target number of bytes the cycle aims to reclaim in the heap.
+- The current available memory in the different parts of the heap.
+
+```xml
+<concurrent-kickoff id="12362" timestamp="2020-10-18T13:35:44.341">
+
+<kickoff reason="threshold reached" targetBytes="239014924" thresholdFreeBytes="33024922" remainingFree="32933776" tenureFreeBytes="42439200" nurseryFreeBytes="32933776" />
+
+</concurrent-kickoff>
+```
+
+**Note:** To analyze specific parts of a cycle, you can search for the elements that mark a specific increment of the cycle. For example, you can search for the <concurrent-global-final> element to locate the final increment of the `gencon` global cycle. See the details of a particular cycle, such as the [`gencon` Global Cycle](./vgclog.md/#global-gc-cycle), to determine the element names for particular STW or concurrent GC increments or operations.
```

Comment:
you may add they initial and final increments can also be identifed by cycle-start/ends (of a specific type), too